### PR TITLE
CVSB-10506 - fix vulnerable dependencies

### PR DESCRIPTION
**Pull request by KHodderDVSA**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "build:copy": "cp package.json package-lock.json build && cd src && find ./**/*.yml | cpio -pdm ../build",
     "build:install": "cd build && npm install --production",
     "test": "BRANCH=local jest --testMatch=\"**/*.unitTest.ts\" --coverage",
+    "package": "cd build && mkdir config && cd config && touch config.yml && cd ../ && cp -r package.json package-lock.json config src/ && cd src && npm ci --production && rm package.json package-lock.json && zip -qr ../../${ZIP_NAME}.zip",
```
Comment (agoncearenco) [inline package.json]: What is this needed for?

Comment (agoncearenco) [reply package.json]: Isn't Jenkins doing this already?

Comment (KHodderDVSA) [reply package.json]: Nathan seems to have added it, I assume for a reason?

Comment (agoncearenco) [reply package.json]: Ok, will approve the PR since @nblore is not available at the moment.

**Pull request merged**